Troubleshooting-Doc/TroubleshootingDoc.adoc

```diff
@@ -0,0 +1,61 @@
+= Schulich Epsilon Software Troubleshooting Document
+
+This document contains a high-level overview of Schulich Epsilon's onboard programs.
+
+== Epsilon-Onboard-Media-Control
+
+=== Overview:
+This program is Schulich Epsilon's onboard media controller.
+The program operates on a raspberry pi and runs a music player that will play songs on the pi inside the car.
+The music player is operated through a touch screen located in the center of the car.
+
+== Epsilon-Dashboard
+=== Overview:
+This program operates Schulich Epsilon's dashboard. 
+It is responsible for displaying all important information regarding the car's systems to the driver. 
+The dashboard program runs on and receives data from the main raspberry pi.
+There are 3 configurations that the dashboard can be run in:
+
+==== Race mode:
+The display mode that is used when the car is racing, the race mode dashboard only occupies one display to conserve power.
+
+==== Display mode:
+The default display mode, shows speed, turn signals etc.
```

Comment:
Speed and turn signals are in both modes. 
`The default display mode runs on both displays, and shows extra information that the race mode omits`